src/components/warning-text/_warning-text.scss

```diff
@@ -26,14 +26,14 @@
 
     min-width: 38px;
     min-height: 35px;
-    margin-top: -19px; // Half the height of the 38px circle
+    margin-top: -20px; // Half the height of the 38px circle
```

Comment:
Nit: This comment is confusing now, could update to say 'half the height of the circle'